app/src/main/java/org/thoughtcrime/securesms/home/search/GlobalSearchAdapterUtils.kt

```diff
@@ -87,9 +87,9 @@ private fun getHighlight(query: String?, toSearch: String): Spannable? {
     return SearchUtil.getHighlightedSpan(Locale.getDefault(), BoldStyleFactory, toSearch, query)
 }
 
-fun ContentView.bindModel(query: String?, model: GroupConversation) {
+fun ContentView.bindModel(query: String?, model: LegacyGroupConversation) {
```

Comment:
All those changes (here plus line 92, 102) went from group to legacy groups. How are groups V2 handled?

Reply:
The model `LegacyGroupConversation` can actually handle the new group so I've changed it back the naming